docs/examples/example_satellite_observations.ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>If you adopt my suggestion from above, then here we would need to replace <code>ds['u_surf']</code> by <code>ds['u']</code> and similar for the meridional component. In that case, to compute MKE, you may have to do something like</p><pre class="ql-syntax" spellcheck="false">KE_uvfilt = 0.5*(u_filtered_simple_fixed_factor**2 + v_filtered_simple_fixed_factor**2).squeeze()
</pre><p>We could also consider renaming <code>KE_uvfilt</code>  to <code>MKE</code>  . I would find that more intuitive, and it is in line with your definitions above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocean-eddy-cpt/gcm-filters/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>